index.html

```diff
@@ -33,57 +34,62 @@
                 <h1>Learning version control using Git</h1>
                 <p>
                     <small>
-                        Created by
+                        Created by:
                             <a href="http://www.SanyamKhurana.com">Sanyam Khurana</a> /
                             <a href="http://twitter.com/ErSanyamKhurana">@ErSanyamKhurana</a> /
-                            <a href="http://github.com/CuriousLearner">@CuriousLearner</a>
+                            <a href="http://github.com/CuriousLearner">@CuriousLearner</a> /
+                            <a href="https://thegeekyway.com">The Geeky Way</a><br>
+                        Edited by:
+                            <a href="https://twitter.com/vipulgupta2048">Vipul Gupta</a> | <a href="https://github.com/vipulgupta2048">@vipulgupta2048</a> | <a href="https://mixstersite.wordpress.com">Mixster</a>
                     </small>
                 </p>
             </section>
             <section id="fr1">
                 <h2>Daily Tasks as Programmer</h2>
                 <p>
-                <ul>
-                    <li class="fragment">Create “Things”</li>
-                    <li class="fragment">Save “Things”</li>
-                    <li class="fragment">Edit “Things”</li>
-                    <li class="fragment">Save “Things” AGAIN!</li>
-                </ul>
+                    <ul>
```

Comment:
Let's use spaces here instead of tab. 

let's use it everywhere.

Reply:
Well, I did convert all tabs to spaces. What indent are we looking at 4 or 2

Reply:
Ideally it would be 2, but if it's 4 everywhere now, let it be 4.

Reply:
Made indentations as 2